doc/release/upcoming_changes/22540.expired.rst

```diff
@@ -0,0 +1,3 @@
+* Passing dtype instances other than the default ones to
+  ``dtype=`` or ``signature=` in ufuncs will now raise a
```

Comment:
"default" could use some clarification. I assume some of the outdate alternate (example?) types are deprecated?

Reply:
Hmmm, tried to make it more specific.  Basically, I still allow the singleton instances (and things that are equivalent to it).  But really, we only enforce the DType (class/type) anyway, and are just generous about allowing those.

Reply:
FWIW the new wording LGTM and matches the terminology (i.e. "canonical") in the relevant NEPs